pkgs/development/tools/oh-my-posh/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "oh-my-posh";
+  version = "8.25.0";
+
+  src = fetchFromGitHub {
+    owner = "jandedobbeleer";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-EQyfuxqznEZUJDw28Bi8+g5vaxCvB0OOS4T+uECkIFM=";
+  };
+
+  vendorSha256 = "sha256-t4FpvXsGVsTYoGM8wY2JelscnlmDzrLMPYk7zGUfo58=";
+
+  prePatch = ''
```

Comment:
Does sourceRoot = "source/src"; work here too?

Reply:
Yep, that works! Good call.